scripts/templates/nexus-repository-ha-operator-certified.clusterserviceversion.yaml

```diff
@@ -211,6 +212,11 @@ spec:
     3. Use the Local Storage Operator to automatically create persistent volumes for your chosen storage class name as documented here: https://docs.openshift.com/container-platform/4.13/storage/persistent_storage/persistent_storage_local/persistent-storage-local.html
     4. Specify the storage class name from step 3 as the value of the `nexusData.storageClass.name` property and also set `nexusData.volumeClaimTemplate.enabled` to true.
 
+    #### Configuring the operator for dynamic persistent volume provisioning
+    * Set the `nexusData.storageClass.name` parameter to a storage class name. This could be one of the default storage classes automatically created in your open shift cluster (e.g. if you're using ROSA) or one that you would like the operator to create.
```

Comment:
```suggestion
    * Set the `nexusData.storageClass.name` parameter to a storage class name. This could be one of the default storage classes automatically created in your OpenShift cluster (e.g., if you're using ROSA) or one that you would like the operator to create.
```